routes/api/team.js

```diff
@@ -113,11 +113,15 @@ module.exports = {
          * @apiErrorExample {object} Error-Response: 
          *      {"message": "Team not found", "data": {}}
          */
-        teamRouter.route("/:id").get(
-            Middleware.Validator.RouteParam.idValidator,
+        teamRouter.route("/").get(
```

Comment:
Placing id in body of `get` breaks the client-side implementation, sadly, because `GET` traditionally does not have any body parameters, only query params. 

https://stackoverflow.com/questions/29571284/for-restful-api-can-get-method-use-json-data

Reply:
https://stackoverflow.com/questions/978061/http-get-with-request-body

Reply:
The big issue is that I'm not able to pass the team's id through route parameters like other get routes due to the way our auth is structured, as there is no way to go from a team id to a specific individual's accountId due to multiple hackers in a team. A possible solution is to put use an identify function as input to ensureAuthorize, and then only have /team/:all for get. What do you think?

Reply:
Yes, that might be better.